src/Samples/NetCoreClient/Program.cs

```diff
@@ -15,14 +21,43 @@ static void Main(string[] args)
             ((IClientChannel)channel).Close();
             factory.Close();
 
-            factory = new ChannelFactory<Contract.IEchoService>(new BasicHttpBinding(), new EndpointAddress("http://localhost:8080/basichttp"));
+            factory = new ChannelFactory<Contract.IEchoService>(new BasicHttpBinding(), new EndpointAddress(_basicHttpEndPointAddress));
             factory.Open();
             channel = factory.CreateChannel();
             ((IClientChannel)channel).Open();
             Console.WriteLine("http Echo(\"Hello\") => " + channel.Echo("Hello"));
             ((IClientChannel)channel).Close();
             factory.Close();
 
+            // 
```

Comment:
Can you refactor the two ways into two methods and have the Main method call each in turn. I'll leave the method naming to you, but as a suggestion, "CallUsingWCF" and "CallUsingWebRequest".

Reply:
Certainly!  I refrained from doing so based on the relevant rules!  (Yes, I'm the one person that did read them!) ;)